### ISSUE TITLE
A question

### ISSUE THREAD

**Issue by lizhiwei98** [jmstadt/DryOilySkin#3]
Are you using a deep learning training model to get a skin classifier？Could you answer me?Thanks!!

**Comment by jmstadt** [comment]
Yes, I used FastAi (on top of Pytorch).  It was a while ago so I was using Fastai v1 (I know there is a newer version of Fastai now).  But, it was basically a variant of this notebook that I followed https://github.com/fastai/course-v3/blob/master/nbs/dl1/lesson1-pets.ipynb

**Comment by lizhiwei98** [comment]
> Yes, I used FastAi (on top of Pytorch). It was a while ago so I was using Fastai v1 (I know there is a newer version of Fastai now). But, it was basically a variant of this notebook that I followed https://github.com/fastai/course-v3/blob/master/nbs/dl1/lesson1-pets.ipynb

Thank you for your quick answer！Do you have a trained model？Or could you provide relevant source code？Thanks！

**Comment by lizhiwei98** [comment]
> Yes, I used FastAi (on top of Pytorch). It was a while ago so I was using Fastai v1 (I know there is a newer version of Fastai now). But, it was basically a variant of this notebook that I followed https://github.com/fastai/course-v3/blob/master/nbs/dl1/lesson1-pets.ipynb


Could you provide the corresponding data set of oily dry skin？I've been looking for a long time, but I can't find the relevant data set.Thanks！

**Comment by lizhiwei98** [comment]
This data set is really hard to find,  could you provide it?    I really, really, really thank you！ Hope your reply.

**Comment by lizhiwei98** [comment]
hello，could you reply me？My teacher assigned me this task. if i cannot finish this task，i donnot kown what’s going to happen.




| |
lizhiwei98
|
|
邮箱：lizhiwei98@126.com
|

Signature is customized by Netease Mail Master

On 03/01/2021 21:42, jmstadt wrote:

Yes, I used FastAi (on top of Pytorch). It was a while ago so I was using Fastai v1 (I know there is a newer version of Fastai now). But, it was basically a variant of this notebook that I followed https://github.com/fastai/course-v3/blob/master/nbs/dl1/lesson1-pets.ipynb
